js/Ice/bidir/MockAlarmClock.ts

```diff
@@ -0,0 +1,30 @@
+
+import { Ice } from "@zeroc/ice";
+import { EarlyRiser } from "./AlarmClock.js";
+
+export class MockAlarmClock extends EarlyRiser.AlarmClock
+{
+    private _needMoreTime = true;
+    private _resolver: ((value: void | PromiseLike<void>) => void);
```

Comment:
Is this a standard name/pattern in JS/TS?

We have a Utils.Resolver in src, but that's not what you're using here.

Reply:
the standard promise takes to callbacks resolve and reject.